### PR TITLE
We don't just list countries, but also regions like EU 

### PR DESCRIPTION
**Pull request by def-**

### CODE REVIEW

File: src/game/client/components/menus_settings.cpp [hunk 1]
```diff
@@ -347,7 +347,8 @@ void CMenus::RenderSettingsPlayer(CUIRect MainView)
 	MainView.HSplitTop(20.0f, 0, &MainView);
 	static float s_ScrollValue = 0.0f;
 	int OldSelected = -1;
-	UiDoListboxStart(&s_ScrollValue, &MainView, 50.0f, Localize("Country"), "", m_pClient->m_pCountryFlags->Num(), 6, OldSelected, s_ScrollValue);
+	str_format(aBuf, sizeof(aBuf), "%s / %s", Localize("Country"), Localize("Region"));
```
Comment (heinrich5991) [inline src/game/client/components/menus_settings.cpp:350]: Always localize the whole string, other languages have other separators, etc. There's no other mention of `Localize("Country")` in the code.

Comment (def-) [reply src/game/client/components/menus_settings.cpp:350]: The problem is that everyone then has to retranslate Country or we fill out the translation half-way, but then translators won't even discover that it's not translated yet.

Comment (def-) [reply src/game/client/components/menus_settings.cpp:350]: @heinrich5991 Thoughts on how to proceed?

**Comment by def-** [comment]
Ready for review.

**Comment by heinrich5991** [comment]
Thanks.

bors r+

**Pull request merged**